packages/webui/src/components/ui/TraceDetail.tsx

```diff
@@ -90,123 +92,131 @@ export default function TraceDetail() {
-      </div>
-
-      <div data-test-id="summary" className="p-default">
-        <div className="flex flex-row">
-          <div className="flex-0 mr-2 md:mr-4">
-            <img src={getIconUri(trace)} alt="" className="w-6 h-6 md:w-12 md:h-12" />
-          </div>
-          <div className="flex-1 flex flex-col">
-            <div className="break-all">
-              <span className="flex justify-between items-center">
-                <span data-test-id="method" className="font-bold">
-                  {method}
-                </span>
-                {responseComplete && statusCode && (
-                  <span data-test-id="status" className="flex items-center gap-2">
-                    <span className={statusCodeStyle(statusCode)}></span>
-                    {`${statusCode > -1 ? statusCode : ''} ${statusMessage}`}
-                  </span>
-                )}
-              </span>
-              <span data-test-id="url" className="block text-secondary">
-                {url}
-              </span>
+    <div className="h-full flex flex-col bg-secondary">
+      <div className="sticky top-0">
+        <div data-test-id="summary" className="p-default mb-4">
+          <div className="flex flex-row">
+            <div className="flex-0 mr-2 md:mr-4">
+              <img src={getIconUri(trace)} alt="" className="w-6 h-6 md:w-12 md:h-12" />
             </div>
-            <div className="flex flex-row flex-wrap justify-between items-center mt-4">
-              <div data-test-id="service">
+            <div className="flex-1 flex flex-col">
+              <div className="break-all">
+                <span className="flex justify-between items-center">
+                  <span data-test-id="method" className="font-bold">
+                    {method}
+                  </span>
+                  <span className="flex items-center gap-2">
+                    <CopyAsCurlButton data-test-id="copy-as-curl" trace={trace} />
+                    <IconButton
+                      Icon={HiX}
+                      onClick={() => clearSelectedTrace()}
+                      className="py-2.5"
+                      data-test-id="close-trace"
+                    />
+                  </span>
+                </span>
+                <span data-test-id="url" className="block">
+                  {url}
+                </span>
+              </div>
+              <div data-test-id="service" className="mt-2">
                 Sent from <span className="font-bold">{serviceName}</span>
               </div>
-              <CopyAsCurlButton data-test-id="copy-as-curl" trace={trace} />
             </div>
           </div>
         </div>
+
+        {responseComplete && statusCode && (
+          <div data-test-id="response-status" className="absolute bottom-0 right-0 p-default flex items-center gap-2">
+            <span className={statusCodeStyle(statusCode)}></span>
+            {`${statusCode > -1 ? statusCode : ''} ${statusMessage}`}
```

Comment:
When this is false, would we end up printing an extra start space here? (The character before `${statusMessage}` inside the TTL)